### DISCUSSION TITLE
Error caused by wagmi installing packages into its own node_modules directory.

### DISCUSSION THREAD

**Discussion by Pavel-Sushko** [wevm/wagmi#1802] [Q&A]
For some reason, the wagmi package creates its own node_modules directory. This causes a TypeError. When I delete the node_modules directory in ./node_modules/wagmi/, I no longer have any error. Why does that happen?

**Comment by tmm** [comment]
Can you post a minimal reproducible example (e.g. package manager, GitHub repo link)?

**Reply by Pavel-Sushko** [reply]
The wrong version of WAGMI was being installed in the package.json, which installed wrong versions of other modules which created the described issue. We have fixed it.
